### ISSUE TITLE
Matlab Data Compatibility

### ISSUE THREAD

**Issue by ccoffrin** [lanl-ansi/PowerModels.jl#134]
All .m files should work with Matlab, but this is not regularly tested;  So errors my arise.  Fixes are always welcome.

The data files may include features that are not supported by Matpower.

Based on the comment in #131, it seems that,
```
	'Area 1'	123	987	'Slack \'Bus\' 1'	1.23	;
```
from `case3.m` does not parse in Matlab because the quotes are not escaped properly?  I was just guessing when I wrote this one.

**Comment by frederikgeth** [comment]
Indeed, we removed the it to "'Slack Bus 1'" for the case3_dc.m as well.

**Comment by ccoffrin** [comment]
It looks like double quote is the correct Matlab escape character?

https://www.mathworks.com/matlabcentral/newsreader/view_thread/31522

If so, I can update the parser to support this.
